### PR TITLE
OBGM-403 Bump a few Gradle dependencies to latest versions

### PR DESCRIPTION
**Pull request by mdpearson**
Minor housekeeping - closing out a stale ticket

### CODE REVIEW

**Comment by awalkowiak** [comment]
@mdpearson could you take your commits and cherry-pick those on top of the new rebased version of https://github.com/openboxes/openboxes/tree/feature/upgrade-to-grails-3.3.10?

**Comment by mdpearson** [comment]
@awalkowiak done

File: build.gradle [hunk 1]
```diff
@@ -438,7 +438,7 @@ dependencies {
     compile 'com.google.code.gson:gson:2.10.1'
 
     // expose javax.annotation.Nullable, com.google.common.base.Enums, etc.
-    compile 'com.google.guava:guava:31.1-jre'
+    compile 'com.google.guava:guava:32.0.0-jre'
```
Comment (awalkowiak) [inline build.gradle:441]:
```suggestion
    compile 'com.google.guava:guava:32.0.1-jre'
```

**Review by awalkowiak — COMMENTED** [review]
Minor security patch bump

**Pull request merged**